exonum-java-binding-core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -60,27 +60,32 @@ impl JavaServiceRuntime {
     /// # Panics
     ///
     /// - If user specified invalid additional JVM parameters.
-    fn create_java_vm(config: JvmConfig) -> JavaVM {
+    fn create_java_vm(jvm_config: JvmConfig, ejb_config: EjbConfig) -> JavaVM {
```

Comment:
This code became too complex to work without documentation and/or extra methods:
```
def buildJvmArguments(jvm_config, ejb_config) -> JvmArgs
  var args_builder = …
  // Prepend extra user arguments
  addUserArguments(args_builder, jvm_config.args_prepend);
  // Add required arguments
  addRequiredArguments(args_builder, ejb_config);
  // Add optional arguments
  addOptionalArguments(args_builder, jvm_config);
  // Append extra user arguments
  addUserArguments(args_builder, jvm_config.args_append);
  return args_builder.build();
```

Reply:
That's actually was my intention as well to rewrite it this way in order to make it testable.